### DISCUSSION TITLE
rAudio App + 7" Pi screen

### DISCUSSION THREAD

**Discussion by beeniedave** [rern/rAudio#1754] [General]
Hello all. 
just discovered rAudio after trying to get other players running without much luck. hope you can help me with a couple of questions.

Tried to download the rAudio App but play store says no; my phone, (a Poco X3 NFC running MIUI 14 / android 12) is not compatible. What phone do I need to use this App?

Can I use the standard 7" RPi screen?  and what settings do I use to enable this?

Many thanks
Dave

**Comment by R101G** [comment]
@beeniedave 
So far as I know, the local screen is enabled by default, although the on-board keyboard does not appear for some fields. The app works, but Android has a problem with it, so you cannot download or update it. Somewhere on this site there is an API version you can use, I think. Otherwise, if you get it on-line, any browser should do if you enter the IP or raudio.local, or whatever your OS needs. If it does not connect, it should appear as a wifi AP, for which information is also on here somewhere.

**Comment by FreshImmuc** [comment]
You should be able to download the file for Android here: https://apkpure.com/raudio/com.raudio/downloading
It should be compatible with Android 6+, try your luck

**Reply by beeniedave** [reply]
Thank you for the link, just got it installed on my phone and appears to be working.  will spend some time learning the different settings and how the system works.
cheers  Dave
